src/System.ServiceProcess.ServiceController/tests/TestServiceProvider.cs

```diff
@@ -102,6 +104,7 @@ private void CreateTestServices()
             string processName = Process.GetCurrentProcess().MainModule.FileName;
             string entryPointName = typeof(TestService).Assembly.Location;
             string arguments = TestServiceName;
+            testServiceInstaller.Username = Username;
```

Comment:
Why not set this above where we're setting all the properties of testServiceInstaller?